glusterd2/peer/peer.go

```diff
@@ -24,3 +26,14 @@ type ETCDConfig struct {
 	ClusterState   string
 	DeletePeer     bool
 }
+
+// GetMetadataSize returns the size of metadata from peer info
+func (p Peer) GetMetadataSize() int {
+	currentSize := 0
```

Comment:
Name this as just `size`

Do the same for other versions of this function for peer, volume and request.